buildscript.gradle

```diff
@@ -15,38 +15,35 @@
  */
 
 rootProject.buildscript {
-  String rosMavenPath = System.getenv("ROS_MAVEN_PATH")
-  String rosMavenRepository = System.getenv("ROS_MAVEN_REPOSITORY")
-
-  repositories {
-    if (rosMavenPath != null) {
-      rosMavenPath.tokenize(":").each { path ->
+    String rosMavenPath = System.getenv("ROS_MAVEN_PATH")
+    String rosMavenRepository = System.getenv("ROS_MAVEN_REPOSITORY")
+    repositories {
+        if (rosMavenPath != null) {
+            rosMavenPath.tokenize(":").each { path ->
+                maven {
+                    // We can't use uri() here because we aren't running inside something
+                    // that implements the Script interface.
+                    url "file:${path}"
+                }
+            }
+        }
         maven {
-          // We can't use uri() here because we aren't running inside something
-          // that implements the Script interface.
-          url "file:${path}"
+            url "http://repository.springsource.com/maven/bundles/release"
         }
-      }
-    }
-    maven {
-      url "http://repository.springsource.com/maven/bundles/release"
-    }
-    maven {
-      url "http://repository.springsource.com/maven/bundles/external"
+        maven {
+            url "http://repository.springsource.com/maven/bundles/external"
+        }
+        if (rosMavenRepository != null) {
+            maven {
+                url rosMavenRepository
+            }
+        }
+        maven {
+            url "https://github.com/rosjava/rosjava_mvn_repo/raw/master"
+        }
+        mavenCentral()
     }
-    if (rosMavenRepository != null) {
-      maven {
-        url rosMavenRepository
-      }
-    } else {
-      maven {
-        url "https://github.com/rosjava/rosjava_mvn_repo/raw/master"
-      }
+    dependencies {
+        classpath "org.ros.rosjava_bootstrap:gradle_plugins:[0.2,0.3)"
     }
-    mavenCentral()
-  }
-
-  dependencies {
-    classpath "org.ros.rosjava_bootstrap:gradle_plugins:[0.2,0.3)"
-  }
 }
```

Comment:
For our information...this makes sure that it always gets set, even on windows when there is no environment variable. When there is, worst case it just sets it twice, best case, it lets a user add a second maven repository to pull from. *More power to the user*.